cfgov/jinja2/v1/_layouts/spanish-ask-base.html

```diff
@@ -1,200 +1,26 @@
-<!DOCTYPE html>
-<!--[if lt IE 7]>      <html lang="es" class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
-<!--[if IE 7]>         <html lang="es" class="no-js lt-ie9 lt-ie8"> <![endif]-->
-<!--[if IE 8]>         <html lang="es" class="no-js lt-ie9"> <![endif]-->
-<!--[if gt IE 8]><!--> <html lang="es" class="no-js"> <!--<![endif]-->
-<head prefix="og: http://ogp.me/ns# fb: http://ogp.me/ns/fb#">
+{% extends '/es/es-base.html' %}
 
-<meta charset="utf-8">
-<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-{% block babel_title %}
-<title>Obtener Respuestas - Oficina para la Protección Financiera del Consumidor</title>
-<meta name="title" content="Obtener Respuestas - Oficina para la Protección Financiera del Consumidor">
-{% endblock %}
-
-{% block page_meta %}
-<meta property="og:title" content="Obtener Respuestas - Oficina para la Protección Financiera del Consumidor">
-<meta property="og:url" content="{{ request.url }}">
-{% endblock %}
-
-<meta property="og:type" content="government">
-<meta property="og:site_name" content="Oficina para la Protección Financiera del Consumidor">
-<meta property="fb:page_id" content="141576752553390">
-<meta property="fb:app_id" content="210516218981921">
-
-<meta name="description" content="">
-<meta name="viewport" content="width=device-width initial-scale=1">
-<meta name="Copyright" content="Public domain">
-<link rel="apple-touch-icon" href="{{ static('nemo/_/img/apple-touch-icon-precomposed.png') }}">
-<link rel="apple-touch-icon" size="72x72" href="{{ static('nemo/_/img/apple-touch-icon-72x72-precomposed.png') }}">
-<link rel="apple-touch-icon" size="114x114" href="{{ static('nemo/_/img/apple-touch-icon-114x114-precomposed.png') }}">
-<link rel="apple-touch-icon" size="144x144" href="{{ static('nemo/_/img/apple-touch-icon-144x144-precomposed.png') }}">
+{%- block title -%}
+    {%- block babel_title -%}
+        Obtener Respuestas - Oficina para la Protección Financiera del Consumidor
+    {%- endblock -%}
+{%- endblock title -%}
 
+{% block og_title %}Obtener Respuestas - Oficina para la Protección Financiera del Consumidor{% endblock %}
 
-<!-- Custom modernizr for touch detection and svg support only, see the .js file for the build link -->
-<script src="{{ static('nemo/_/js/modernizr.custom.js') }}"></script>
-
-<!--[if lt IE 9]>      <link rel="stylesheet" href="{{ static('nemo/_/c/es-styles-ie.min.css') }}"> <![endif]-->
-    <!--[if gt IE 8]><!--> <link rel="stylesheet" href="{{ static('nemo/_/c/es-styles.min.css') }}"> <!--<![endif]-->
-
-<!--[if lt IE 9]>      <link rel="stylesheet" href="{{ static('knowledgebase/es-ask-styles-ie.min.css') }}"> <![endif]-->
+{% block css %}
+    {{ super() }}
+    <!--[if lt IE 9]>      <link rel="stylesheet" href="{{ static('knowledgebase/es-ask-styles-ie.min.css') }}"> <![endif]-->
```

Comment:
Is this right? So if it's greater than IE9 this will resolve to:

```
<!--      <link rel="stylesheet" href="{{ static('knowledgebase/es-ask-styles-ie.min.css') }}"> <!-->
```

Seems like the closing HTML comment at the beginning is missing.

Reply:
Nevermind, I guess I'm forgetting how these resolve https://www.quirksmode.org/css/condcom.html